pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -170,6 +181,23 @@ contract ManagedPool is BaseWeightedPool, ReentrancyGuard {
         // Validate and set initial fee
         _setManagementSwapFeePercentage(params.managementSwapFeePercentage);
 
+        // Set initial value of the protocolSwapFeePercentage; can be updated externally if it is delegated
+        if (_DELEGATE_PROTOCOL_FEES_SENTINEL == params.protocolSwapFeePercentage) {
```

Comment:
Hard-wired into my brain this is, from C/C++, where you had to always put the constant on the left to avoid inadvertent assignment.
 
 That's no longer a problem, so change it I will, for consistency with the rest of the codebase